### ISSUE TITLE
RPGLE full free highlighting break

### ISSUE THREAD

**Issue by Didiv** [barrettotte/vscode-ibmi-languages#19]
![image](https://user-images.githubusercontent.com/7267354/78655803-2acf1800-78cf-11ea-8a5b-fdb38ecc00f1.png)

```
**free
/copy mpkcorsrc,pr_f3gtrtd
dcl-s RLBERRCNT packed(3);
dcl-s i like(RLBERRCNT);
```

**Comment by barrettotte** [comment]
Thanks for reporting, I believe that https://github.com/barrettotte/vscode-ibmi-languages/issues/15
should resolve this and will be included in version 0.5.1 of the extension that I will be releasing within a day or two

**Comment by barrettotte** [comment]
Fixed with https://github.com/barrettotte/vscode-ibmi-languages/issues/15
and released in v0.5.1 of the extension
